### DISCUSSION TITLE
Вебхук при проведении платежа

### DISCUSSION THREAD

**Discussion by p-sanachev** [sberdevices/salute-issues#223] [SmartPay]
Есть ли какой-то вебхук, который можно перехватить своим сервером после проведения платежа?

**Comment by awinogradov** [comment]
В рамках сценария? Или где?

**Reply by p-sanachev** [reply]
SmartPay через SmartApp API. 
То есть - делается Canvas App, за фронтом есть какой-то бэк. 
Вопрос в том, есть ли в рамках SmartPay через API не только опрос о статусе транзакции, но и какой-то вебхук

**Reply by awinogradov** [reply]
Документация про это есть тут https://developer.sberdevices.ru/docs/ru/payment_tools/smartpay_api#%D0%BF%D1%80%D0%BE%D0%B2%D0%B5%D0%B4%D0%B5%D0%BD%D0%B8%D0%B5-%D0%BF%D0%BB%D0%B0%D1%82%D0%B5%D0%B6%D0%B0

Пример реализации на JS тут 
https://github.com/sberdevices/salutejs/blob/master/examples/payment-chatapp/src/index.ts#L26
https://github.com/sberdevices/salutejs/blob/master/examples/payment-chatapp/src/handlers.ts#L130
